packages/jobs/lib/processor/handler.ts

```diff
@@ -65,10 +65,10 @@ async function action(task: TaskAction): Promise<Result<JsonValue>> {
         return Err(error);
     }
     const res = jsonSchema.safeParse(response);
-    if (res.success) {
-        return Ok(res.data);
+    if (!res.success) {
+        return Err(`Invalid action response format: ${response}. Action: ${JSON.stringify(task)}`);
```

Comment:
is it safe to stringify the task/reponse in term of credentials and such?

Reply:
You are right. It contains the action input which contains customers' data. I will change to return the id only. That's the minimum for debugging